pyseir/cdc/utils.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+
+class Target(Enum):
+    CUM_DEATH = 'cum death'
+    INC_DEATH = 'inc death'
+    CUM_HOSP = 'cum hosp'
+    INC_HOSP = 'inc hosp'
+
+
+class ForecastTimeUnit(Enum):
+    DAY = 'day'
+    WK = 'wk'
```

Comment:
Minor: I think `WEEK` is more readable than `WK`, but that's personal preference

Reply:
Ah this is a required unit by CDC, I agree week would better : )